iina/Base.lproj/MainMenu.xib

```diff
@@ -30,7 +30,7 @@
                                     <action selector="showAboutWindow:" target="Voe-Tx-rLC" id="HGv-E1-lQH"/>
                                 </connections>
                             </menuItem>
-                            <menuItem title="Check for Updates..." id="gMo-qL-m08">
+                            <menuItem title="Check for Updates…" id="gMo-qL-m08">
```

Comment:
There should be a space between the text and the ellipsis character! Short explanation: Ellipsis directly behind a word means the word is truncated. Ellipsis withe leading space means that if you click the menu-item, you will be taken to a dialogue instead of immediately executing a command. So many people di that wrong. Can someone fix that, i don't have the time to deal with that. Thanks!